exonum/src/runtime/dispatcher/migration_tests.rs

```diff
@@ -173,6 +177,18 @@ fn complex_migration_part2(ctx: &mut MigrationContext) -> Result<(), MigrationEr
     Ok(())
 }
 
+fn not_good_then_ok(_ctx: &mut MigrationContext) -> Result<(), MigrationError> {
+    static SUCCESS: AtomicBool = AtomicBool::new(false);
```

Comment:
Nit: `static`s and atomics could be avoided if `MigrationRuntime` would have a flag denoting whether to return a "good" script or a "bad" one for `"not-good-then-ok"` migration.